src/components/swimlanes/swimLanes.tsx

```diff
@@ -11,94 +11,95 @@ import {
   Link,
 } from "@nypl/design-system-react-components";
 import styles from "./Swimlanes.module.css";
-import React from "react";
 import { imageURL } from "@/utils/utils";
 import appConfig from "appConfig";
+import FeaturedContentComponent from "../featuredContent/featuredContent";
 
 const SwimLanes = ({ lanesWithNumItems }) => {
-  return (
-    <>
-      {lanesWithNumItems.map((lane, key) => (
-        <Box data-testid={lane.slug} key={lane.rank} mt="xxl">
-          <Flex alignItems="baseline">
-            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
-              {lane.title}
-            </Heading>
-            <Spacer />
-            <Link
-              id={`row-see-more-${lane.slug}`}
-              type="standalone"
-              href={`${appConfig.DC_URL}/collections/lane/${lane.slug}`}
-              aria-label={`See more ${lane.title.toLowerCase()}`}
-              sx={{
-                display: { sm: "none", md: "inline" },
-                fontWeight: "500",
-                alignItems: "center",
+  let hasRendered = false;
```

Comment:
What if you tried inserting the `FeaturedContent` component into the `swimLanesElements` array through the `.splice` function? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

If that works, you could move the array insertion outside of this component so that `swimLanes.tsx` only returns swimlanes and not the featured content.